src/components/add-mfa-method/add-mfa-methods-controller.ts

```diff
@@ -0,0 +1,29 @@
+import { Request, Response, NextFunction } from "express";
+import { PATH_DATA } from "../../app.constants";
+
+export function addMfaMethodGet(req: Request, res: Response): void {
+  const helpText = `<p>${req.t("pages.addMfaMethod.app.help.text1")}</p><p>${req.t("pages.addMfaMethod.app.help.text2")}</p>`;
+  res.render(`add-mfa-method/index.njk`, { helpText });
+}
+
+export function addMfaMethodPost(
+  req: Request,
+  res: Response,
+  next: NextFunction
+): void {
+  const { addMfaMethod } = req.body;
+
+  if (addMfaMethod === "sms") {
```

Comment:
Can "sms" be a constant?  (Same for "app" below).

Reply:
Done, and refactored a bit to use the constants more widely